pkg/component/observability/plutono/dashboards/seed/shoot-operation-duration.json

```diff
@@ -0,0 +1,309 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Plutono --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "",
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "id": 1,
+  "iteration": 1734701406787,
+  "links": [],
+  "panels": [
+    {
+      "collapsed": false,
+      "datasource": null,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 10,
+      "panels": [],
+      "title": "Shoot Durations",
+      "type": "row"
+    },
+    {
+      "datasource": "seed-prometheus",
+      "description": "The average time it takes for shoot $operation_type operation.",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "percentage",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              }
+            ]
+          },
+          "unit": "short"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 9,
+        "w": 24,
+        "x": 0,
+        "y": 1
+      },
+      "id": 4,
+      "options": {
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "showThresholdLabels": false,
+        "showThresholdMarkers": false,
+        "text": {}
+      },
+      "pluginVersion": "7.5.34",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "rate(gardenlet_shoot_operation_duration_seconds_sum{operation=~\"$operation_type\",workerless=~\"$workerless\",hibernated=~\"$hibernated\"}[$__rate_interval]) / rate(gardenlet_shoot_operation_duration_seconds_count{operation=~\"$operation_type\",workerless=~\"$workerless\",hibernated=~\"$hibernated\"}[$__rate_interval])",
```

Comment:
Typically, dashboards either allow selecting a percentile variable out of a few options (like the [Controller Runtime / Controllers](https://github.com/gardener/gardener/blob/b6b857890afabfa10c75a2016f3f02ddde5762a6/pkg/component/observability/plutono/dashboards/common/controllers.json) dashboard) or visualize three percentiles next to each other over time (like the [Controller Runtime / Controller Details](https://github.com/gardener/gardener/blob/master/pkg/component/observability/plutono/dashboards/common/controller-details.json) dashboard).

Can you do either option on this dashboard?